lab_1_classify_by_unigrams/main.py

```diff
@@ -64,7 +91,14 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    squa_diff = 0
+    if len(actual) == len(predicted):
+        for i, x in enumerate(actual):
+            squa_diff += (x - predicted[i]) ** 2
+    else:
+        return None
+    mse = round((squa_diff/len(actual)), 4)
```

Comment:
do not create the variable - return `as is`

Reply:
done, it seems

Reply:
no

Reply:
done, it seems